deploy/kubernetes/base/controller-deployment.yaml

```diff
@@ -31,7 +31,7 @@ spec:
         - name: efs-plugin
           securityContext:
             privileged: true
-          image: "amazon/aws-efs-csi-driver:v1.1.1"
+          image: "amazon/aws-efs-csi-driver:latest"
```

Comment:
we don't push latest tag anymore, so latest is probably ponting to some super old version.

I don't think it matters since our README instructs to use the stable overlay...installing the base templates as-is, naked, is probably not a good idea in any case.

Reply:
So I'll update it to v1.2.0 here?